### DISCUSSION TITLE
Bibliography without In-text Citations?

### DISCUSSION THREAD

**Discussion by brutaldigital** [thegetty/quire#555] [Get Help]
Hi there,

Is it possible to display a list of bibliographic references at page-level using `q-bibliography` without using the `q-cite` in-text citations? If so, where and how should I reference the entries from the `references.yml`  file?

Thanks!
Tom

**Comment by Erin-Cecele** [comment]
Hi @brutaldigital. You have a few options: 

1. You can either use `reference.yml` and `q-bibliography` to create an overall bibliography at the end of the publication 
2. You can add a page-level biblio simply using Markdown
3. You can add the references to a `references.yaml` file and then list the relevant `ids` in the page YAML. However, this may be a bit unwieldy if there are a lot of references. See example of page YAML below. 
 ```
title: Chapter Title
order: 10
layout: page
references:
  - "West 1939"
  - "Evans 1938"
  - "Lynd 1929"
 ``` 
The order they’re listed in doesn’t matter. Quire will sort them all alphabetically. Although it's worth noting that you may need to add a `sort` attribute to your `references.yml` if diacritics or other exceptions knock things out of order. More info on this can be found in our documentation: https://quire.getty.edu/docs-v1/citation-bibliographies/.

**Comment by Erin-Cecele** [comment]
Hi @brutaldigital, did our answer help resolve your issue? Also, out of curiosity, are you currently working on a publication for Quire, or are you just in an exploration phase with the tool?

**Comment by brutaldigital** [comment]
Thanks @Erin-Cecele . I'm just in an exploration phase for the moment.

I'm trying out your suggestion to add a page-level biblio in markdown at the bottom of the page, but am struggling with the order. No matter how I arrange the markup the notes come after after the bibliography content (and not below their corresponding heading). Is there a way to avoid this, or must footnotes always be the last section on a page?

> {% backmatter %}
> ## Notes
> [^...]
> ## Bibliography
> [...]
> {% endbackmatter %}

**Reply by Erin-Cecele** [reply]
Hi @brutaldigital. Ah, yes, Markdown processors are designed to always put Notes last. In this case, the best thing to do would be to make use of option 3 listed above (use the `references.yaml` file and then refer to those `ids` in the page YAML of the essay). Due to a bug, this function is not currently available in Quire v1. We are bringing this to the attention of our developers and hope to have a fix for you soon. (I will link the GH issue number shortly).

**Reply by Erin-Cecele** [reply]
@brutaldigital You can now track our progress with this bug in issue #654.
